packages/core/src/part.js

```diff
@@ -416,7 +416,20 @@ Part.prototype.setCutOnFold = function (p1, p2) {
 }
 
 Part.prototype.isEmpty = function() {
```

Comment:
I'm still on the fence about this. I remember you said that you can't trust designs to have accurately set `render=false` but I feel that we should fix the underlying issue, rather than paper over it with this check.